lms/urls.py

```diff
@@ -434,6 +434,15 @@
             url(r'^courses/{}/teams'.format(settings.COURSE_ID_PATTERN), include('teams.urls'), name="teams_endpoints"),
         )
 
+    if settings.FEATURES.get('ENABLE_MOBILE_XBLOCKS'):
```

Comment:
LTI has its own public endpoints (`lti_provider_launch` and `lti_provider_run`) to this same underlying implementation.  So that URL will continue to be available.

However, you have a good point that this particular endpoint is generic and not specific to Mobile.  The reason I put it behind a Mobile-specific feature-flag is since it will be reviewed as part of other endpoints that are being implemented for Mobile xBlocks.  

Hmm.. ok.  I can rename this feature flag.  How about "ENABLE_RENDER_XBLOCK_API"?